src/styles/components/_title.scss

```diff
@@ -5,11 +5,11 @@
 @import "base/variables";
 
 .homepage_section_block {
+	background-color: $color-pink;
```

Comment:
weird spacing here? please align all properties, do you have a SASS plugin installed in vscode?

Reply:
will align! def having sass plugin.. spacing might come from copy-paste